### ISSUE TITLE
Essential feature request: option to pin to every virtual desktop

### ISSUE THREAD

**Issue by Lakmal98** [LorenzCK/OnTopReplica#109]
In windows 10, virtual multiple desktop is a very useful feature. So, While working with more than one virtual desktop this pinned window is only in one desktop. It is more useful to pinned this window for every virtual desktop.

**Comment by SuperKael** [comment]
You can do this yourself using a built-in Windows feature. Press WindowsKey+Tab, right click on the OnTopReplica window, and select "Show windows from this app on all desktops". Simple as that.

**Comment by LorenzCK** [comment]
I agree, this is something that should be enabled by default. Also, OnTopReplica should be able to discriminate between windows open on the current virtual desktop and windows on other ones.
